### PR TITLE
Gather current release versions into global variables.

### PR DESCRIPTION
**Pull request by mtbc**
Gathers the various release-version values together from the Virtual Microscope and SLS Gallery playbooks, replacing any "latest" with the currently installed version.

### CODE REVIEW

**Comment by mtbc** [comment]
Once this is in I'll open PRs for upgrade to OMERO 5.6.

File: learning.yml [hunk 1]
```diff
@@ -119,9 +115,9 @@
 
     - role: ome.omero_web_apps
       omero_web_apps_packages:
-        - omero-gallery==3.1.1
-        - omero-iviewer==0.8.1
-        - omero-virtual-microscope==1.0.1
+        - omero-gallery=={{ omero_web_apps_release["omero_gallery"] }}
```
Comment (manics) [inline learning.yml]: FYI if you have plain variable names you can also use `{{ omero_web_apps_release.omero_gallery }}`

Comment (joshmoore) [reply learning.yml]: Leaving that to the follow up.

Comment (mtbc) [reply learning.yml]: :+1:

**Pull request merged**